packages/rest-typings/src/v1/dm/im.ts

```diff
@@ -53,7 +53,7 @@ export type ImEndpoints = {
 
 	'/v1/im.members': {
 		GET: (params: DmMemberProps) => PaginatedResult<{
-			members: Pick<IUser, '_id' | 'status' | 'name' | 'username' | 'utcOffset'>[];
+			members: (Pick<IUser, '_id' | 'status' | 'name' | 'username' | 'utcOffset'> & { isMuted: boolean })[];
```

Comment:
```suggestion
			members: Pick<IUser, '_id' | 'status' | 'name' | 'username' | 'utcOffset'>[];
```
Since DM members can't be muted, I'm not sure if we should have this new field here. I guess the `im.members` and `dm.members` endpoints haven't been changed also, so rolling back this change should be fine